supervisor/api/const.py

```diff
@@ -10,3 +10,7 @@
 ATTR_USE_NTP = "use_ntp"
 ATTR_USE_RTC = "use_rtc"
 ATTR_APPARMOR_VERSION = "apparmor_version"
+ATTR_UPDATE_PATH = "update_path"
```

Comment:
The easiest way to keep it compatible and future proof is by having the supervisor host the "Update available" page and have that page do the update. In that case we don't need `update_path`.